2008-01-26  Jakub Jelinek  <jakub@redhat.com>

	* gcc.c-torture/compile/pr34966.c (atan): Only use asm
	on i?86/x86_64.

--- gcc/testsuite/gcc.c-torture/compile/pr34966.c.jj	2008-01-26 09:55:35.000000000 +0100
+++ gcc/testsuite/gcc.c-torture/compile/pr34966.c	2008-01-26 10:00:22.000000000 +0100
@@ -4,7 +4,11 @@ __inline double
 atan (double __x)
 {
   register double __result;
+#if defined(__i386__) || defined(__x86_64__)
   __asm __volatile__ ("" : "=t" (__result) : "0" (__x));
+#else
+  __result = __x;
+#endif
   return __result;
 }
 
